hudi-common/src/main/java/org/apache/hudi/keygen/constant/KeyGeneratorType.java

```diff
@@ -97,9 +99,12 @@ public enum KeyGeneratorType {
   SPARK_SQL_MERGE_INTO("org.apache.spark.sql.hudi.command.MergeIntoKeyGenerator"),
 
   @EnumFieldDescription("A test KeyGenerator for deltastreamer tests.")
-  STREAMER_TEST("org.apache.hudi.utilities.deltastreamer.TestHoodieDeltaStreamer$TestGenerator");
+  STREAMER_TEST("org.apache.hudi.utilities.deltastreamer.TestHoodieDeltaStreamer$TestGenerator"),
```

Comment:
Could `STREAMER_TEST` be removed, since this is intended as `USER_PROVIDED` key generator type? Adjust the corresponding test to use `USER_PROVIDED` key generator type.